### DISCUSSION TITLE
setting default time

### DISCUSSION THREAD

**Discussion by AnotL** [Stellarium/stellarium#2758] [Q&A] [answered]
I downloaded the latest version to a PC.  In Configuration, in the Time tab...it's not showing the time and date for my location.  And I'm not planning on moving this computer, so I wanted the default to remain the same.   

In the Time tab it's showing the date and time as 1-12-1999 18:00.  Why is that?  I can change it, but when I close the program and restart it, that date appears again.

I'm not sure what is meant by, System date at: with the time 10:00 PM shown.

And, on the tool bar, the date is correct, but the time shown is one hour less than my actual local time.  It indicates UTC -5, but the time shown is actually UTC -6.  

Thanks.

**Comment by gzotti** [comment, marked as answer]
Have you checked the Location window? Here you can disable the automatic lookup of your location from the network (which may occasionally fail), set your timezone properly, and disable daylight saving time. 

In the Configuration dialog, Time tab, you can select whether the program starts up with current system time or some preconfigured date. The option "System date at: 10:00 PM" means "TONIGHT at 10 PM". "System" in this context means your computer. 

After changing options, don't forget to press "Save Settings" in the Configuration dialog to actually save settings. See User Guide, chapters 3+4 for a thorough introduction.
